pkgs/top-level/all-packages.nix

```diff
@@ -28874,18 +28874,6 @@ in
 
   solfege = python3Packages.callPackage ../misc/solfege { };
 
-  disnix = callPackage ../tools/package-management/disnix { };
```

Comment:
Do we need aliases for them in aliases.nix?

Reply:
done